src/Makefile

```diff
@@ -33,3 +47,7 @@ pvAccess_SYS_LIBS_WIN32 += ws2_32
 
 include $(TOP)/configure/RULES
 
+# Can't use EXPAND as generated headers must appear
+# in O.Common, but EXPAND emits rules for O.$(T_A)
+../O.Common/pvaVersionNum.h: pvaVersionNum.h@
+	$(EXPAND_TOOL) $(EXPANDFLAGS) $($@_EXPANDFLAGS) $< $@
```

Comment:
> There is a subtle difference in definitions. I have to say I find
> 
>     EXPAND_VARS += EPICS_PVA_MAJOR_VERSION="$(EPICS_PVA_MAJOR_VERSION)"
> 
> a little verbose.

Ah, didn't spot that little refinement. Of course you could have used
shorter variable names in the local Makefile, say MAJOR, MINOR, MAINT
and, er, DEVMT say.

* Andrew

-- 
There are only two hard problems in distributed systems:
  2. Exactly-once delivery
  1. Guaranteed order of messages
  2. Exactly-once delivery
 -- Mathias Verraes